tcf_website/management/commands/load_grades.py

```diff
@@ -67,14 +67,19 @@ def clean(self, df):
         CourseGrade.objects.all().delete()
         CourseInstructorGrade.objects.all().delete()
 
-        return df.dropna(
+        # Filter out data with no grades
+        df = df.dropna(
```

Comment:
Not really part of this PR per se, but same thing with `qs` as before; honestly not sure what general practice is, but I personally think we should avoid variable names that are basically just letters.

Reply:
I believe using `df` for any pandas `DataFrame` instance is very commonplace, and this is a function with very limited scope, so I don't think there's value in using a descriptive name for this parameter.